test/ui/test_spec.js

```diff
@@ -96,29 +94,42 @@ describe('UI tests', function() {
     });
 
     it('appears correctly before any actions have been taken', function(done) {
-      return screenshotAndCompare('body.png', 'body', done);
+      screenshotAndCompare('body.png', 'body')
+        .then(function() {
```

Comment:
Even if you don't choose to collapse all the one-liners, I think you can still write all these calls to done as
  .then(done);